packages/babel-plugin/src/utils/build-compiled-component.ts

```diff
@@ -142,3 +142,58 @@ export const buildCompiledComponent = (
 
   return compiledTemplate(node, sheets, meta);
 };
+
+/**
+ * Accepts a cloneElement node and returns a Compiled Component AST.
+ *
+ * @param node Originating cloneElement node
+ * @param cssOutput CSS and variables to place onto the component
+ * @param meta {Metadata} Useful metadata that can be used during the transformation
+ */
+export const buildCompiledCloneElement = (
+  node: t.CallExpression,
+  cssOutput: CSSOutput,
+  meta: Metadata
+): t.Node => {
+  const { sheets, classNames } = transformCssItems(cssOutput.css, meta);
+
+  const props = node.arguments[1];
+
+  // TODO: This is a temporary fix to prevent the plugin from crashing when the second argument of cloneElement is not an object expression.
+  if (!t.isObjectExpression(props)) {
+    throw new Error('Second argument of cloneElement must be an object expression.');
+  }
```

Comment:
If someone is using a variable/spread/etc it'll probably be impossible to reliably fish out and modify the CSS prop. 
Are there any standard approaches to this problem in use in the repo already? Do we throw/Log an err/try really hard to locate the variable?